src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -874,6 +874,8 @@ def _get_all_job_states(self, job_ids=None):
                 state['run_id'] = self._running_jobs[job_id]['job'].run_id
                 if status == 'finished':
                     self._completed_job_states[state['job_id']] = dict(state)
+                state['job_input'] = literal_eval(state.get('job_input', '{}'))
```

Comment:
This has bothered me for a few days, and I think I finally figured out why - the API shouldn't be returning goofy non-standard JSON strings that require an AST to eval, but the actual unpacked structure. I think that should be revisited.

Reply:
👍 updated ee2 API.